lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -1236,4 +1236,54 @@ private function isNumericType(Type $type) : bool
     {
         return $type instanceof IntegerType || $type instanceof BigIntType;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getDropIndexSQL($index, $table = null)
```

Comment:
There's a lot of added code in this child class - are we 100% sure that all of it is covered by test automation?

Reply:
Yes